.storybook/main.ts

```diff
@@ -2,38 +2,31 @@ import type { StorybookConfig } from "@storybook/react-webpack5";
 import { storybookAddonStylingWebpackConfig } from "./addon-styling-webpack-config";
 
 const config: StorybookConfig = {
-  features: {
-    storyStoreV7: true,
-  },
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
   staticDirs: ["../static"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-a11y",
     "@storybook/addon-interactions",
-    "storybook-addon-performance",
     "@storybook/addon-coverage",
     {
       name: "@storybook/addon-styling-webpack",
       options: storybookAddonStylingWebpackConfig,
     },
+    "@storybook/addon-webpack5-compiler-babel",
+    "@chromatic-com/storybook",
```

Comment:
Can this chromatic visual testing add-on `@chromatic-com/storybook` be removed, as we don't tend to bundle this as it has associated costs

Reply:
Done!